Basic/ClientDriven/Packages/manifest.json

```diff
@@ -5,7 +5,7 @@
     "com.unity.ide.visualstudio": "2.0.16",
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",
-    "com.unity.netcode.gameobjects": "1.0.1",
+    "com.unity.netcode.gameobjects": "1.0.2",
```

Comment:
shouldn't it be 1.1.0?

Reply:
This is true. I did it through the package manager and it didn't pull in the latest. Will update!

Reply:
yeah I encountered the same thing while updating the other samples. I get a message saying 1.1.0 is not verified for 2021.3.12f1, but I suppose this is simply because this version is still new